x/pool-incentives/spec/02_state.md

```diff
@@ -0,0 +1,28 @@
+<!--
+order: 2
+-->
+
+# State
+
+### Genesis states
+
+```go
+type GenesisState struct {
+	// params defines all the paramaters of the module.
+	Params            Params          
+	LockableDurations []time.Duration 
+	DistrInfo         *DistrInfo      
+}
+
+type Params struct {
+	// minted_denom is the denomination of the coin expected to be minted by the minting module.
+	// Pool-incentives module doesn’t actually mint the coin itself, but rather manages the distribution of coins that matches the defined minted_denom.
+	MintedDenom string 
+	// allocation_ratio defines the proportion of the minted minted_denom that is to be allocated as pool incentives.
+	AllocationRatio github_com_cosmos_cosmos_sdk_types.Dec 
+}
+```
+
+Lockable durations can be set to the pool incentives module at genesis. Every time a pool is created, the `pool incentives` module creates the same amount of 'pot' as there are lockable durations for the pool.
```

Comment:
Why do we need this module to create pots?  Can't we just expect that someone creates a pot themselves, before submitting a proposal to add a pot to incentives?

Also, then we don't need to have predefined LockableDurations, right?  DistrRecords only care about PotIDs, not durations, right?

Reply:
I also don't see a need to creates the pot automatically, I think this will just cause a lot more state bloat for us in distribution logic. I think its fine to require the user to create the pot they want to incentivize. (Or roll it into one conjoined CLI command)

Reply:
Pots can be created by sending a transaction.

I don't think it's a big deal to have it happen now, although in the future, we may want to remove this.

Created an issue for this:  #187